### DISCUSSION TITLE
Device Identification

### DISCUSSION THREAD

**Discussion by praveshgupta1586** [talsec/Free-RASP-Flutter#116] [General]
Dear Team,
We integrated "freerasp" in our application and we are getting reports also. However we are not able to identify the device id from our system with shared report by "freerasp". Our application generates a seprate device ID using vendorID in iOS and Settings.Secure.ANDROID_ID in android. But the " freerasp" reports show different device IDs so we are unable to identify the effected devices.
Please help us to identify the device from report.

**Comment by msikyna** [comment]
https://github.com/orgs/talsec/discussions/35
